lib/knapsack_pro/queue_allocator.rb

```diff
@@ -15,61 +15,49 @@ def initialize(args)
 
     def test_file_paths(can_initialize_queue, executed_test_files)
```

Comment:
@3v0k4 This method gives tests to run to Knapsack. It handles Queue Mode.

I like the structure so far. It has a straight flow and guard clauses.

There is also a file `lib/knapsack_pro/test_suite_builder.rb`, which is something new.

Let's sync on a call to see what we like or not before polishing this more.